### ISSUE TITLE
RFC: simplify about screen

### ISSUE THREAD

**Issue by pazos** [koreader/koreader#7743]
Two proposals:

1. Replace these two menu entries:

Version (on press shows extended version)
About (on press shows about infomessage)

with one named "KOReader v2021.xx" that shows the about message on press.

2. Some minor style changes in the about message:

### current
![current](https://user-images.githubusercontent.com/975883/119267990-32e03900-bbf1-11eb-9462-11b09a4f54b4.png)

### proposed
![new](https://user-images.githubusercontent.com/975883/119267994-35429300-bbf1-11eb-826a-9ac6e23a6989.png)

Short version ought to be enough for users here. The full version is still available under "report a bug".

**Bonus points** for making the link actionable if `Device:canOpenLink()`

**Comment by Frenzie** [comment]
I'm not a big fan of the number only version. Unless you have any tips for how to reduce that to an actual commit. The date and the hash are the ones that are useful to me. But that aside.

**Comment by NiLuJe** [comment]
What @Frenzie said, although, admittedly, if it's still visible in Report-a-bug, I'm okay with that :).

**Comment by pazos** [comment]
> I'm not a big fan of the number only version. Unless you have any tips for how to reduce that to an actual commit. The date and the hash are the ones that are useful to me. But that aside.

Is the hash enough or you want the date too?

![new](https://user-images.githubusercontent.com/975883/119494578-c0db3180-bd61-11eb-8a7d-680ce090c4bc.png)

**Comment by Frenzie** [comment]
I don't care about the date much but most users do.

**Comment by Frenzie** [comment]
But I wrote Version to help with this stuff. It could just say the date on the next line or something. 

http://koreader.rocks/doc/modules/version.html#Version:getBuildDate

It doesn't need to be the exact string.

**Comment by Frenzie** [comment]
Oops, sorry, that's the git-rev modification date. But either way, something like that could be an idea.
